modules/dnn/test/test_layers.cpp

```diff
@@ -742,6 +742,18 @@ TEST_P(Test_Caffe_layers, Average_pooling_kernel_area)
     normAssert(out, blobFromImage(ref));
 }
 
+TEST_P(Test_Caffe_layers, PriorBox_repeated)
+{
+    Net net = readNet(_tf("prior_box.prototxt"));
+    Mat inp = blobFromNPY(_tf("priorbox_input_1.npy"));
+    Mat shape = blobFromNPY(_tf("priorbox_input_2.npy"));
```

Comment:
I think we can just generate inputs - actual values do not matter for PriorBox layer.